Header marissa

msrissaxox opened the pull request:
- Removed node modules from being tracked

msrissaxox commented:
- Added header component and removed extra assets and pictures that were automatically installed with project

codebyangi commented:
Hi. I just review this. It looks great, but before merging, maybe we can add the name SurgiTrack (I believe we got a few likes on this one) instead of the current name. Also, visually, I would prefer to add the same color as we have on the footer, for consistency reasons only. What do you think? Thank you

msrissaxox commented:
I added the name and also changed the background color to match the footer. Thank you!

> Hi. I just review this. It looks great, but before merging, maybe we can add the name SurgiTrack (I believe we got a few likes on this one) instead of the current name. Also, visually, I would prefer to add the same color as we have on the footer, for consistency reasons only. What do you think? Thank you
